### ISSUE TITLE
BUG - Screen update fails

### ISSUE THREAD

**Issue by jthoel** [Snorby/snorby#259]
Using Linux (CentOS 6.3), Chrome Version 23.0.1271.91

To replicate.. 
- Go to events, select and event and then do 'Snorby Search by Source' or Dest on either address..   you get a list of events with that IP
- Try to select a few, using the checkbox and apply a classification.
- The screen ghosts, but you never get the 'marking events' message at the top and the ghosting never goes away. 
- If you refresh the URL (click in the URL box and hit enter) the same query comes back and you can see that the events are marked.

So basically it's not doing a proper full update on the status of the alerts, why the alerts are derived from a specific query.
